### ISSUE TITLE
Program Runs and Outputs Data but Networks do not die

### ISSUE THREAD

**Issue by HitItAll** [DanMcInerney/wifijammer#69]
I am testing this program on Kali Nethunter with an external wifi card that supports monitor mode, but when I launch the program (With No Parameters) it outputs data while not affecting the wifi networks.
Any help would be appreciated, and please tell me if you need further info.

**Comment by M1ND-B3ND3R** [comment]
How are you sure the wireless networks arent being affected?

**Comment by ll3N1GmAll** [comment]
I am experiencing this same issue. Tested on a couple different distros (all pen testing distros) with this same issue. All MACs in range keep echoing to the screen as though the program was working; but no clients or APs that I am testing against are affected at all. I do not even see any speed issues which would indicate a client was deauthed momentarily. I have also tested this on ubuntu after having installed all dependencies and am getting almost totally blank output on the screen as though it is running; but doesn't see any MACs. Hardware supports injection and monitor mode. Tested this with 2 different NICs as well. No change in performance. Why would this program run and not have any effect on APs in range? If it is sending out deauths then APs should be impacted.
